demo/helloclient.R

```diff
@@ -8,6 +8,7 @@ library(grpc)
 
 spec <- system.file('examples/helloworld.proto', package = 'grpc')  
 impl <- read_services(spec)
+options(client_deadline = 10)
```

Comment:
You already have a 5 seconds deadline in the backbone code making this a bit redundant if not weird. The client will fail with a deadline more than 5 seconds, yes but this should probably be in your personal tests and not in the final code.

Reply:
Actually, I don't get why is it weird?
Let me break it into simple detailed steps,
1. Before the update the main problem was that RPC calls have a client-side deadline of 5 seconds.
2. This was a problem because some RPCs require more than 5 seconds.
3. After adding the client_deadline argument you will be able of doing the following
      - Set your own desired client-side deadline i.e. the time at which the client-side may timeout.
      - If this time is not set by the user it will be automatically set into 5 seconds.

So, I don't exactly get what is "5 seconds deadline in the backbone" that you are referring to?
There is no more hard-coded time in the underlying wrapper. Now, if you set the client_deadline to 10 seconds, then the client-side will timeout in case RPC requires more than 10 seconds. It will not timeout before that. I hope that you get what I'm saying and if you didn't I will be pleased to elaborate more.

Regarding the test, in my opinion I think it is essential to guide the user of this feature in the provided examples so that he can use it in future implementations. It will not affect the main code either way but I think it is important to include it in at least one of the provided examples, but of course it is up to Neal to decide whether it would be important or not to merge it to the current code.

Reply:
Okay, you are guiding the user by providing an example. It looked to me like you had forgot to remove some of the code that you had used to test.

Reply:
Well yes I was trying to provide a simple example showing the feature. That's why I didn't exactly get what were you saying. However, everything is explained in the commit description you can check it if you like!